utils/build/docker/nodejs/express4-typescript/app.ts

```diff
@@ -191,14 +191,20 @@ app.all('/tag_value/:tag/:status', (req: Request, res: Response) => {
   }
 
   res.status(parseInt(req.params.status) || 200).send('Value tagged');
```

Comment:
weird that it works
```suggestion
  res.status(parseInt(req.params.status) || 200)
```

Reply:
It is working, but it is also failing :-D

In this line and in line 204 we are sending the same response. The first send works and sends 200 to the response, the second send fails because the response is already closed, but the test already has the needed information. The application logs the error and thats all.

The send of line 202, is for `payload_in_response_body`, which is a feature that it is not implemented in node yet, so the failure in this test is expected and it works.

Running the test locally I'm having this error in stderr.log.

```
Error: Cannot set headers after they are sent to the client
    at __node_internal_captureLargerStackTrace (node:internal/errors:484:5)
    at new NodeError (node:internal/errors:393:5)
    at ServerResponse.setHeader (node:_http_outgoing:644:11)
    at ServerResponse.setHeader (/usr/app/node_modules/dd-trace/packages/datadog-instrumentations/src/http/server.js:84:41)
    at ServerResponse.header (/usr/app/node_modules/express/lib/response.js:794:10)
    at ServerResponse.send (/usr/app/node_modules/express/lib/response.js:174:12)
    at /usr/app/dist/app.js:168:13
    at /usr/app/node_modules/dd-trace/packages/datadog-instrumentations/src/router.js:50:25
    at /usr/app/node_modules/dd-trace/packages/datadog-shimmer/src/shimmer.js:26:21
    at Layer.handle [as handle_request] (/usr/app/node_modules/express/lib/router/layer.js:95:5)
```